### ISSUE TITLE
Warning: count(): Parameter must be an array or an object that implements Countable

### ISSUE THREAD

**Issue by diit** [bayareawebpro/illumine-framework-starter#1]
Love the direction of the project! Just ran into a few issues.

Installed the plugin and activated after `composer install`, no modification.

## Expected: 
Installs correctly and activates.

## Actual:
`Warning: count(): Parameter must be an array or an object that implements Countable in /var/www/html/wp-content/plugins/emterra-product-tracker/vendor/bayareawebpro/illumine-framework/Plugin.php on line 42`

**Comment by bayareawebpro** [comment]
@diit thank you for the feedback!  I spent the last few months rebuilding the framework based on Lumen instead as it's more lightweight.  I'll push out a new version once I deploy the plugin I'm developing into the wild, should be done this week.   I removed some of the utilities in this version so it's slimmed down.  I also fixed a few bugs with shortcode rendering.  I removed the ObjectBuffering and got it to work without it.  Stay tuned!
